spec/unit/accordion/template.js

```diff
@@ -0,0 +1,9 @@
+var template = [
```

Comment:
An interesting post-MVP idea to explore here: have this file be a standard HTML file or handlebars file. This would bring us closer to having a components HTML separated with the component, in the same way [fractal](https://github.com/frctl/fractal) does.